parquet/src/file/page_index/index.rs

```diff
@@ -70,11 +65,9 @@ where
 
 #[derive(Debug, Clone, PartialEq)]
 #[allow(non_camel_case_types)]
-/// Typed statistics for a data page in a column chunk.
+/// Statistics for data pages in a column chunk.
 ///
-/// This structure is part of the "Page Index" and is optionally part of
-/// [ColumnIndex] in the parquet file and can be used to skip decoding pages
-/// while reading the file data.
+/// See [NativeIndex] for more information
```

Comment:
Not familiar with Rust doc by L:30 is [`NativeIndex`], this line is [NaiveIndex], do they have any differences?

Reply:
the rendered slightly differently in the generated rustdoc ([`NativeIndex`] renders in a monospace font)

It would be good to keep them consistent. Thank you for the catch

```suggestion
/// See [`NativeIndex`] for more information
```